example/90-shoot.yaml

```diff
@@ -335,8 +335,6 @@ spec:
     # authenticationMode: basic # allowed values: basic,token
 # tolerations:
 # - key: <some-key>
-# Explicitly specify the seed that will run the shoot control plane.
-# seedName: my-seed
```

Comment:
Instead of removing it, can you adapt the comment to say that end-users (or users without `update shoots/binding` permissions) are not allowed to specify it?
You could reference the scheduler / `shoots/binding` documentation